simplified-ui-navigation-tabs/src/main/java/org/librarysimplified/ui/navigation/tabs/TabbedNavigator.kt

```diff
@@ -70,7 +70,16 @@ class TabbedNavigator private constructor(private val navigator: BottomNavigator
   }
 
   fun popBackStack(): Boolean {
-    return this.navigator.pop()
+    if (this.navigator.currentStackSize() < 1) {
+      return false
```

Comment:
Does the `this.navigator.pop()` already handle this? I imagine it doesn't, so it would be helpful to add a log here too so we can be aware if it's possible to call this even when there's nothing to pop. Also, you could add this to the `try-catch`

Reply:
This was just pure paranoia. I think it crashes instead due to an internal bug.

Reply:
Adjusted!